sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/tests/ServiceBusEndToEndTests.cs

```diff
@@ -194,6 +194,13 @@ public async Task TestBatch_Messages()
             await TestMultiple<ServiceBusMultipleMessagesTestJob_BindToMessageArray>();
         }
 
+        [Test]
+         public async Task TestBatch_MinBatchSize()
```

Comment:
Can we add a test where the min batch size isn't reached before the MaxWaitTime? We should still see that the function gets invoked by the MaxWaitTime (with some buffer).